client/modules/IDE/components/PreviewFrame.jsx

```diff
@@ -419,7 +401,7 @@ PreviewFrame.propTypes = {
   textOutput: PropTypes.bool.isRequired,
   gridOutput: PropTypes.bool.isRequired,
   soundOutput: PropTypes.bool.isRequired,
-  setTextOutput: PropTypes.func.isRequired,
+  // setTextOutput: PropTypes.func.isRequired,
```

Comment:
should this be removed rather than commented out?

Reply:
Will do.